src/components/data/listing/TableView.js

```diff
@@ -286,10 +292,15 @@ function TableView(props) {
                             listing.map((resource, index) => {
                                 const resourceName = resource.label;
                                 const resourceId = resource.id;
+                                const resourcePath = `${path}/${resource.label}`;
                                 const isSelected =
                                     selected.indexOf(resourceId) !== -1;
                                 const isInvalid =
                                     isSelected && isInvalidSelection(resource);
+                                const [instantLaunch] = defaultInstantLaunch(
+                                    instantLaunchDefaultsMapping,
+                                    resource
+                                );
```

Comment:
This returns the instant launch object that is marked as the default for the resource, or null if there isn't one. If we ever want to support having multiple instant launches possible for a file, this is one of the calls that will end up changing.